MaxText/integration/tunix/tunix_adaptor.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import annotations
```

Comment:
could you use two spaces for indentation (use `bash code_style.sh`) to follow MaxText style.

Any plans for tests for this adapter?

Reply:
updated spaces.

Discussed offline about plans of further tests